ports/silabs/README.md

```diff
@@ -115,18 +114,18 @@ At the boot stage, two scripts will be run (if not booting in safe mode). First,
 
 After code.py has finished executing, a REPL prompt will be presented on the serial port. Other files can also be executed by using the **Thonny** editors or using **Ampy** tool.
 
-![Thony](./res/Thony.png)
+![Thonny](./res/Thonny.png)
 
 With the boards which support USB mass storage, we can drag the files to the board file system. However, because the EFR32 boards don’t support USB mass storage, we need to use a tool like **Ampy** to copy the file to the board. You can use the latest version of **Ampy** and its  command to copy the module directories to the board.
 
-Refer to the guideline below for installing the **Ampy** tool: 
+Refer to the guide below for installing the **Ampy** tool:
 
-https://learn.adafruit.com/micropython-basics-load-files-and-run-code/install-ampy  
+<https://learn.adafruit.com/micropython-basics-load-files-and-run-code/install-ampy>
 
-## Modules supported ##
+## Built-in modules available ##
```

Comment:
an up to date listing similar to this automatically generated at build time, on a dedicated page: https://docs.circuitpython.org/en/latest/shared-bindings/support_matrix.html -- would it make more sense to remove this table? It feels like a potential maintenance burden to have it here.

Reply:
This was added in the original SiLabs PR, I have no objections for removing it.

Reply:
Done!